explore/src/main/scala/explore/ExploreLayout.scala

```diff
@@ -127,7 +127,7 @@ object ExploreLayout:
                       .toAsync
                 case _                        => IO.unit
               }
-          ): (ExploreEvent => IO[Unit])
+          ).compile.drain.unsafeRunAndForget()(IORuntime.global)
```

Comment:
This is not great, but apparently lucuma-broadcast-channel was also just using `IORuntime.global` for its callback.

In any case, I am fairly certain there should be no memory leak on unmount, because the only reference to this fiber would be via the callback, so long as the callback holder is GCed the this fiber can be as well.

If there is some way to attach the lifecycle of a `Resource[IO, Unit]` to the lifecycle of a component then that's ideally what we should do.

Reply:
`crystal` has some useful hooks for `useStream*` and 'useResource*`

Reply:
It seems like here we probably want `useStreamOnMountBy`.

Reply:
Yup, that should do the trick. You will have an extra useless parameter in the next hooks/render though. I wonder if we want a `useEvalStream` (or something like that) for this kind of situations.

Reply:
Sure. We do the same think in `CacheComponent`. I'll work with what I have for now.